build.zig

```diff
@@ -25,96 +28,114 @@ pub fn build(b: *Builder) !void {
 
     const fmt_step = b.addFmt(&[_][]const u8{
         "build.zig",
+        "rt.zig",
```

Comment:
Perhaps rt.zig could be moved to the test dir?

Reply:
could do